functions-and-scope.js

```diff
@@ -14,6 +14,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+
+let highGrades = 0
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >=8) {
+        highGrades = highGrades + 1;
```

Comment:
Dit kun je nog iets korter schrijven
```suggestion
        highGrades++;
```